file-attribute-caching/src/main/kotlin/com/pkware/filesystem/attributecaching/AttributeCachingPath.kt

```diff
@@ -210,6 +226,11 @@ internal class AttributeCachingPath(
             target.basicAttributes = basicFileAttributes
             target.dosAttributes = dosFileAttributes
             target.posixAttributes = posixFileAttributes
+            target.aclFileAttributeView = if (supportedViews.contains("acl")) {
```

Comment:
This could just be `target.aclFileAttributeView = aclFileAttributeView`, as it's nullable like the rest.

Reply:
this follows the same check logic above on lines 217-223 for the dos and posix file attributes to check if the view is supported

Reply:
yeah i went with your shortened version ryan and added the supportedFileViews lazy attribute, I think I did it the original way to save other_iops hits with calling delegate.fileSystem.supportedFileAttributeViews() once instead of for each lazy attribute that was null.